dpdd.tex

```diff
@@ -828,6 +826,31 @@ \subsubsection{Crowded Field Photometry}
 
 Note that this discussion only pertains to processing of \emph{direct images}. Crowding is not expected to significantly impact the quality of data products derived from \emph{difference images} (i.e., Prompt products).
 
+\subsubsection{Shear Measurement}
+\label{sec:shearMeasurement}
+
+Cosmology is one of the major pillars of LSST science, and a substantial fraction of the survey's cosmological constraining power will come from weak gravitational lensing, which involves estimating the shear field applied to a population of galaxies by foreground matter.
+
+For our pixel-level measurements of galaxy shapes to be usable for shear estimation, the responses of the algorithms we apply to shear must be carefully controlled and/or measured.
+While the state of the art for these algorithms continues to evolve, our plan for at least DR1 and DR2 is to use the \texttt{Metadetection} algorithm of \citet{2020ApJ...902..138S}.
+Additional algorithms may be run as well, as long as their processing time and catalog storage costs are negligible (which is actually the case for a some promising but still unproven methods, e.g. \citet{2016MNRAS.459.4467B}).
+
+This is a variant of the \texttt{Metacalibration} \citep{2017arXiv170202600H,2017ApJ...841...24S} approach, in which the response to shear is measured by applying the same algorithms to sheared versions of the original (coadd) images and their PSF models, and then using finite differences to compute the derivative of those measurements with respect to shear.
+\texttt{Metadetection} requires the algorithms run on these images to include detection and deblending, and like all variants of \texttt{Metacalibration}, it requires all algorithms run on the modified images to be simple enough for finite differences to work well.
+In practice this means measurements must be as close as possible to linear in the pixel values.
+This is incompatible with our usual desire for algorithms to produce catalogs that map as well as possible to astrophysical reality, especially since theway in which the coadd images are modified represents a slight degredation in signal-to-noise.
+
+As a result, the measurements from \texttt{Metadetection} are best structured as a separate \texttt{ShearObject} table representing a completely distinct set of detections and measurements, with many fewer columns than the main \Object table and approximately 5$\times$ as many rows (for the modified versions of the original coadds).
```

Comment:
We could say 3-5 times as many rows.  It isn't clear yet we need to do counterfactual shears in both e1 and e2

Reply:
Done.